packages/ui/src/watch/header/notifications.test.tsx

```
@@ -0,0 +1,223 @@
+    }));
+
+    vi.mocked(useQueryContext).mockImplementation(() => ({
+      notifications: {
+        data: [],
+        isLoading: false,
+        refetch: vi.fn(),
+      },
+      videos: { data: [], isLoading: false, refetch: vi.fn() },
+      posts: { data: [], isLoading: false, refetch: vi.fn() },
+    }));
+  });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)